src/me.rs

```diff
@@ -83,7 +84,12 @@ const fn get_mv_range(
     * (8 * MI_SIZE) as isize
     + border_h;
 
-  (mvx_min, mvx_max, mvy_min, mvy_max)
+  (
```

Comment:
I believe it's possible to jump motion vectors off of adjacent motion vectors/global motion. It would be nice to document this when are restricting the range of all motion vectors.

Reply:
I'd like to get clarity on this constraint, particularly to confirm if it is normative in the spec. I will confirm which version of libaom I was looking at tomorrow.

Reply:
The relevant logic in libaom begins here: https://aomedia.googlesource.com/aom/+/v2.0.0/av1/decoder/decodemv.c#1101
Note that `read_mv()` sums the delta and reference in the output: https://aomedia.googlesource.com/aom/+/v2.0.0/av1/decoder/decodemv.c#878
The `is_mv_valid()` test is applied to the output: https://aomedia.googlesource.com/aom/+/v2.0.0/av1/decoder/decodemv.c#1206

Finally, found the relevant part of the spec: https://aomediacodec.github.io/av1-spec/#assign-mv-semantics